components/Navbar.js

```diff
@@ -79,9 +101,14 @@ const StyledDiv = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
-  border-radius: 20px;
-  width: 46px;
-  height: 21px;
+  border-radius: 100%;
+  width: 35px;
+  height: 35px;
   background-color: ${props =>
     props.active === true ? "rgba(255, 123, 137)" : ""};
 `;
+
+const StyledButton = styled.button`
+  background-color: transparent;
+  border: none;
+`;
```

Comment:
Here you have another button with exactly the same settings as in `Header.js`. Please consider moving reusable components into a separate file.

Reply:
done!